### PR TITLE
Feature ETP-753: Process Definition Action buttons

### PR DESCRIPTION
**Pull request by Sankku**
This implementation pass to the toolbar the button structure needed for the frontend

### CODE REVIEW

File: src/com/etendoerp/metadata/Utils.java [hunk 1]
```diff
@@ -4,13 +4,23 @@
...
+import org.openbravo.service.json.DataResolvingMode;
+import org.openbravo.service.json.DataToJsonConverter;
 import org.openbravo.service.json.JsonUtils;
+import org.openbravo.model.ad.datamodel.Column;
+
 
 import javax.servlet.http.HttpServletRequest;
 import java.io.BufferedReader;
 import java.io.IOException;
+import java.util.HashMap;
```
Comment (LuuchoRocha) [inline src/com/etendoerp/metadata/Utils.java:23]: what is the reason of this changes?

File: src/com/etendoerp/metadata/builders/TabBuilder.java [hunk 2]
```diff
@@ -276,11 +286,13 @@ private JSONObject createProcessJSON(Process process) throws JSONException {
...
 
-    private boolean isProcessField(Field field) {
+    protected boolean isProcessField(Field field) {
         Column column = field.getColumn();
 
-        return column != null && column.getOBUIAPPProcess() != null;
-    }
+        return column != null &&
+                column.getReference() != null &&
+                "28".equals(column.getReference().getId()) &&
```
[the remark this answers is not in the document]
Comment (LuuchoRocha) [reply src/com/etendoerp/metadata/builders/TabBuilder.java:294]: and check if there is possible that a field is some kind of process and uses a diferent reference

**Review by LuuchoRocha — CHANGES_REQUESTED** [review]

File: src/com/etendoerp/metadata/Utils.java (hunk 1 above)
Comment (Sankku) [reply src/com/etendoerp/metadata/Utils.java:23]: Mb, the ProcessUtil was initially here and forgot to remove the imports
